src/app/config.rs

```diff
@@ -280,6 +283,7 @@ pub const ENV_CONTROL_EXP_BACKOFF_MIN: &str = "LINKERD2_PROXY_CONTROL_EXP_BACKOF
 pub const ENV_CONTROL_EXP_BACKOFF_MAX: &str = "LINKERD2_PROXY_CONTROL_EXP_BACKOFF_MAX";
 pub const ENV_CONTROL_EXP_BACKOFF_JITTER: &str = "LINKERD2_PROXY_CONTROL_EXP_BACKOFF_JITTER";
 pub const ENV_TAP_DISABLED: &str = "LINKERD2_PROXY_TAP_DISABLED";
+pub const ENV_TAP_SVC_NAME: &str = "LINKERD2_PROXY_TAP_SVC_NAME";
```

Comment:
I think this should maybe be `..._TAP_SVC_IDENTITY`, rather than `..._TAP_SVC_NAME`, since (IIRC) this is the _service account_ of the tap service rather than the service _name_?

Reply:
I went with `..._TAP_SVC_NAME` since that the pattern we used for `..._DISCOVERY_SVC_NAME` and `..._IDENTITY_SVC_NAME`

Reply:
Upon further investigation, turns out I'm wrong about this; the `DESTINATION_SVC_NAME` and `IDENTITY_SVC_NAME` are expected to be service account names, not service names, so this env var should follow that pattern. 👍